src/filter.js

```diff
@@ -22,8 +22,8 @@ var keys = require('./keys');
  * @category List
  * @sig Filterable f => (a -> Boolean) -> f a -> f a
  * @param {Function} pred
- * @param {Array} filterable
- * @return {Array}
+ * @param {Array} Filterable
+ * @return {Array} Filterable
```

Comment:
Isn't the thing after the `{…}` meant to be the parameter name rather than the type?